app/scripts/lib/setupSentry.js

```diff
@@ -320,6 +320,10 @@ export const SENTRY_UI_STATE = {
     pendingTokens: false,
     welcomeScreenSeen: true,
     useSafeChainsListValidation: true,
+    ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+    addSnapAccountEnabled: false,
+    snapsAddSnapAccountModalDismissed: false,
```

Comment:
what's `snapsAddSnapAccountModalDismissed` for?

Reply:
The first time you install an account Snap, you get a modal displayed to explain what the feature is.

Reply:
ok cool 👍